src/Microsoft.Azure.WebJobs.Host/Bindings/FunctionBindingContext.cs

```diff
@@ -48,5 +52,39 @@ public class FunctionBindingContext
         /// The short name of the current function. 
         /// </summary>
         public string MethodName { get; private set; }
+
+        /// <summary>
+        /// The service provider for the current function invocation scope.
+        /// </summary>
+        public IServiceProvider FunctionServices { get; set; }
```

Comment:
nit -- should this be `InvocationServices` or `InstanceServices` (which we use elsewhere)?

Reply:
I debated. In this context, I ended up landing on this name, but we can use the same for consistency.